pytket/pytket/circuit/__init__.py

```diff
@@ -49,6 +49,8 @@
 # Add ability to compare Bit equality with arbitrary class
 Bit.oldeq = Bit.__eq__
 
+KwargTypes = Union[Bit, int, float, str, None]
```

Comment:
Why this set of types?

Reply:
removed, it is not possible to set the types any more in this way with the direct handover to the c++ kwargs